docs/Language/C++/C++.md

```diff
@@ -9,9 +9,135 @@ has_children: true
+
+- Polymorphism: C++ supports both compile-time polymorphism (function overloading, operator overloading) and runtime polymorphism (virtual functions and abstract classes).
+
+- Data Abstraction and Encapsulation: The ability to hide the implementation details and provide an interface for the user is facilitated through data abstraction and encapsulation.
+
+- Templates: C++ includes a powerful template mechanism, allowing developers to write generic code that works with different data types.
+
+- STL (Standard Template Library): The STL provides a collection of template classes and functions, including algorithms, containers, and iterators, making it easier to write efficient and reusable code.
+
+- Exception Handling: C++ supports exception handling to manage runtime errors more effectively.
+
+- Memory Management: C++ allows manual memory management through new and delete operators, providing control over memory allocation and deallocation.
+
+- Operator Overloading: Operators can be overloaded to work with user-defined data types.
+
+-----------------------------------------------------------------------
+
+### Key Releases
+
+- C++98: The first standardized version of C++, introduced in 1998, included many features like the Standard Template Library (STL), exception handling, and the bool data type.
+
+- C++03: This was a bug-fix release that corrected some issues found in C++98.
+
+- C++11: Released in 2011, C++11 introduced significant features, including auto keyword, range-based for loops, lambda expressions, smart pointers, and the introduction of the "nullptr."
+
+- C++14: Released in 2014, C++14 brought incremental improvements and bug fixes, such as enhanced generic programming features and improvements to the standard library.
+
+- C++17: Released in 2017, C++17 introduced features like structured bindings, std::optional, std::variant, parallel algorithms, and a few language and library enhancements.
+
+- C++20: Released in 2020, C++20 included major features such as concepts (which allow template constraints), ranges library, coroutines, and improvements to existing features like constexpr.
+
+Each release has aimed to enhance the language's expressiveness, safety, and performance, while also improving the overall developer experience. As of my last knowledge update in January 2022, C++23 was under development, but I don't have specific details about its features and changes. It's always a good idea to check the latest documentation and releases for the most up-to-date information.
+
+
+
+-----------------------------------------------------------------------
+
+### Getting Started
+
+Using C++ involves several steps, from setting up your development environment to writing, compiling, and running your code. Here's a step-by-step guide to get you started:
+
+1. ### Install a C++ Compiler
```

Comment:
Not sure if this is an issue referenced locally or " 
![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/47317248/b1a5e9e3-8c33-4ac4-a393-6a18b4a0ec82)
 
The Numbers seem to be showing  1. 1. 1. For some reason.

Reply:
Doesn't seem to be an issue locally. 
![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/43964243/d5c82b39-e683-46d9-97ec-ea10d4216346)

Reply:
Note that if you view the Raw file, I've manually typed 1., 2., 3., etc

Reply:
Might be that github automatically thinks that each one of them is starting a new " Ordered List " and that is why it show as such. Strange issue, have experienced that myself occasionally